data.php

```diff
@@ -156,11 +156,12 @@ function readInBlockList() {
         $file="/etc/pihole/gravity.list";
         $linecount = 0;
         $handle = fopen($file, "r");
-        while(!feof($handle)){
+        if(gettype($handle) == "resource") {
```

Comment:
OK, I will test this out later on today. 

The `fopen` section was something I hacked together from various stack exchange answers (I'm not a PHP developer :smile), but it was the fastest way of reading `gravity.list`!